scripts/nbconverted/integrate-msigdb.py

```diff
@@ -0,0 +1,398 @@
+
+# coding: utf-8
+
+# # Create a hetnet of MSigDB genesets
+# 
+# This script was modified from https://github.com/dhimmel/integrate.
+# 
+# ## For the automatic interpretation of gene expression compression features
+# 
+# The script creates an MSigDB hetnet as described in the eLIFE publication _"Systematic integration of biomedical knowledge prioritizes drugs for repurposing"_ by [Himmelstein et al. 2017](https://doi.org/10.7554/eLife.26726)
+
+# In[1]:
+
+
+import os
+import csv
+import pandas as pd
+import seaborn as sns
+
+import hetio.hetnet
+import hetio.readwrite
+import hetio.stats
+
+
+# In[2]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# ## Define the metagraph and instantiate the graph
+
+# In[3]:
+
+
+kind_to_abbev = {
+    
+    # metanodes
+    'Gene': 'G',
+
+    'Cancer-Hallmarks': 'H',
+    'Positional-Gene-Sets': 'C1',
+    'Curated-Gene-Sets': 'C2',
+    'Motif-Gene-Sets': 'C3',
+    'Computational-Gene-Sets': 'C4',
+    'GO-Gene-Sets': 'C5',
```

Comment:
You may want to split C5 into three metanodes (one for each GO Domain). We've found cellular components are less informative of micro-data-bio-logy than MFs or BPs.

Reply:
Sounds good. I think splitting out other sub-collections would work well too.